iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/bicep/AbstractDeclaration.java

```diff
@@ -44,4 +44,12 @@ protected AbstractDeclaration(SyntaxToken keyword, Identifier identifier, Syntax
   public List<Tree> children() {
     return List.of(keyword, identifier, equals, expression);
   }
+
+  public SyntaxToken keyword() {
+    return keyword;
+  }
+
+  public Identifier identifier() {
+    return identifier;
+  }
```

Comment:
I would prefer to have these getters represented in an interface, as we may want to make the tree API public. There could be a common interface `Declaration`